### PR TITLE
CSS tweaks for add-on config screen.

### PR DESCRIPTION
**Pull request by mrstegeman**

### CODE REVIEW

**Comment by mrstegeman** [comment]
@hobinjk A lot of this is whitespace. https://github.com/mozilla-iot/gateway/pull/813?w=1

File: static/css/addons-form.css [hunk 1]
```diff
@@ -1,171 +1,204 @@
...
+}
+
+.addons-form > p > .form-group > fieldset > legend {
+  font-size: 2.5rem;
+  border-bottom: none;
+  padding-bottom: 2rem;
+}
+
+.addons-form > p > .form-group > fieldset > legend:before {
+  content: 'Configuring: ';
```
Comment (hobinjk) [inline static/css/addons-form.css]: Is it possible to do this in HTML instead?

Comment (mrstegeman) [reply static/css/addons-form.css]: Not easily. The form is completely auto-generated, and the top-level header is generated the exact same way as every nested header.

Comment (hobinjk) [reply static/css/addons-form.css]: Okay, SGTM then

**Pull request merged**